### PR TITLE
Add DigitalOcean one-click docs

### PR DESCRIPTION
**Pull request by derrickmehaffy**
#### Description of what you did:

Adding documentation for the DO one-click customization

~~**This is a WIP but do feel free to provide feedback while I work on it**~~

### CODE REVIEW

**Comment by derrickmehaffy** [comment]
@lauriejim for reference these docs are being constructed to allow for easy migration to the new docs layout (based on your recent changes to the docs-v2 branch)

**Comment by derrickmehaffy** [comment]
@alexandrebodin @lauriejim ready for review. I have tried to keep it as short and simple as possible and just give context to the running config until a documentation writer is brought on.

I could dive a lot deeper into customization but at this point I'm not sure if it's something we should include or not.

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -197,6 +197,7 @@ module.exports = {
             '/3.0.0-beta.x/guides/update-version',
             '/3.0.0-beta.x/guides/databases',
             '/3.0.0-beta.x/guides/deployment',
+            '/3.0.0-beta.x/guides/digitalocean',
```
Comment (lauriejim) [inline docs/.vuepress/config.js]: I think we have to delete this link.

**Comment by derrickmehaffy** [comment]
@lauriejim Link removed

**Pull request merged**